src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -81,12 +70,7 @@
     @Advanced
     @Argument(fullName = "indelSizeToEliminateInRefModel", shortName = "ERCIS", doc = "The size of an indel to check for in the reference model", optional = true)
     public int indelSizeToEliminateInRefModel = 10;
-    
-    /**
-     * Bases with a quality below this threshold will reduced too the minimum usable qualiy score (6).
-     */
-    @Argument(fullName = "base_quality_score_threshold", shortName = "bqst", doc = "Base qualities below this threshold will be reduced to the minimum (" + QualityUtils.MIN_USABLE_Q_SCORE + ")", optional = true)
-    public byte BASE_QUALITY_SCORE_THRESHOLD = PairHMM.BASE_QUALITY_SCORE_THRESHOLD;
```

Comment:
You're right.  A question that then arises is the different defaults between `Mutect2` (`HOSTILE`) and `HaplotypeCaller` (`CONSERVATIVE`).  If it moves to `LikelihoodEngineArgumentCollection` they will share a default, so one tool would need to specify its previous default on the command line.  I don't consider this a big deal, but do you know if it makes much of a difference if M2 is run with the "wrong" `pcrErrorMode`?